tests/h/search/query_test.py

```diff
@@ -4,11 +4,205 @@
 import datetime
 import pytest
 import webob
+import mock
 
 from h.search import Search, index, query
+from hypothesis import strategies as st
+from hypothesis import given
+
+
+MISSING = object()
+ES_VERSION = (1, 7, 0)
```

Comment:
The version is out of date. We could remove `Builder.__init__`'s `es_version` parameter entirely now though.